reactive/ovn_chassis_charm_handlers.py

```diff
@@ -73,6 +73,22 @@ def enable_openstack():
         charm_instance.assess_status()
 
 
+@reactive.when(OVN_CHASSIS_ENABLE_HANDLERS_FLAG)
+@reactive.when_none('charm.installed')
+def maybe_disable_mlockall():
+    with charm.provide_charm_instance() as charm_instance:
+        if charm_instance.options.mlockall_disabled:
```

Comment:
should this have a is_container() check?

Reply:
It is, the call to the `charm_instance.options.mlockall_disabled` property is a call to:
https://github.com/openstack-charmers/charm-layer-ovn/blob/399ce6f4eb8200ccab6f26a282a3840e75a0690f/lib/charms/ovn_charm.py#L82-L95

Reply:
Oh right, I was confusing the method with the charm config option. At least they are word scrambled and not the same name! :P  LGTM